data/ideas.json

```
@@ -185,6 +185,12 @@
         "level": "advanced",
         "tags": "roadmap, success, process"
     },
+    {
+        "title": "Picture Finder",
+        "description": "A website where we search for any picture that is available on unsplash. we can use unsplash API for this, also we can use axios for fetching the data and a UI library for styling.",
+        "level": "beginner",
```

The level should be one of this list  `basic, intermediate, advanced`
```suggestion
        "level": "basic",
```